LICENSE.md

```diff
@@ -1,8 +1,3 @@
-Copyright (c) Connexta
- 
-This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License 
-as published by the Free Software Foundation, either version 3 of the License, or any later version. 
- 
-This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
-<http://www.gnu.org/licenses/lgpl.html>.
\ No newline at end of file
+Copyright (c) 2019 Connexta, LLC
+
+Released under the GNU Lesser General Public License version 3; see https://www.gnu.org/licenses/lgpl-3.0.html
```

Comment:
❓ Why does this not include the full license in the files?

Reply:
I copied this from transforms repo so not sure. I think this is ok though.